packages/grafana-ui/src/components/FileDropzone/FileListItem.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { FileListItem } from './FileListItem';
+
+const file: any = {
```

Comment:
can we create a file in the same way as in `FileDropzone.test.tsx`: https://github.com/grafana/grafana/pull/36646/files#diff-a67ed601a65ccc1680e4978f24b9187a57d39aae05db0fe4e4af89753f774d7cR5? that way we avoid the `any` 😱